joinmarket-webui/jam
# fix: create non-descriptor wallet

Makes sure the created wallet is a non-descriptor wallet.

Changes:
- apply same changes as in https://github.com/joinmarket-webui/jam-docker/pull/58
- replace nbxplorer with own "regtest-initializer"
- update bitcoin core from v22 to v23
- add new npm script `npm run regtest:build` (make use of docker cache)

## How to test
If you just run `npm run regtest:up` you will see that the primary container will start successfully, but the secondary container will fail. This is because if you have not rebuilt your local dev environment (without changes from https://github.com/joinmarket-webui/jam-docker/pull/58), the secondary container defaults to creating a descriptor wallet and fails (which is exactly what is fixed in this PR).

Since https://github.com/joinmarket-webui/jam-docker/pull/58 is already merged, all you have to do is building the images and then start as usual.
1. `npm run regtest:build`
2. `npm run regtest:up`
3. `npm run regtest:init`
4. `npm start`
